app/src/main/res/layout/city_layout.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity"
+    android:background="@color/light_green_color">
+    <TextView
+        android:id="@+id/wecolme_text_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_top_18dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        style="@style/welcome_text"
+        android:text="@string/welcome_text"
+        />
+
+    <ImageView
+        android:id="@+id/weather_image_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toBottomOf="@id/wecolme_text_view"
+        android:layout_marginTop="@dimen/margin_top_18dp"
```

Comment:
Pls, rename the dimen´s because the number at the end its not good way, if you need in a future change the values, you must update the value and rename the dimen.

Reply:
+1

Reply:
Done!